lib/configure/api_users.rb

```diff
@@ -0,0 +1,33 @@
+module Configure
+  class ApiUsers
+    def initialize(namespace:, resource_name_prefix:)
+      @namespace = namespace
+      @name_prefix = resource_name_prefix
+    end
+
+    def configure!(api_users)
+      api_users.each do |api_user|
+        email = [namespace, "#{api_user.fetch('slug')}@alphagov.co.uk"].compact.join("-")
```

Comment:
`alphagov.co.uk` - weird to see this domain - is this a legacy thing that signon already uses? Does it actually send emails to or from these addresses?

Reply:
This domain was the original domain for the development of GOV.UK. For legacy reasons, we still use it for a few things, [list here](https://docs.google.com/document/d/1qdYVIU8dI_CqmwYwwxSfORHEPH-9gX8m0cAJOZcwVOs/edit#heading=h.axp1xnscuvsv).

I've changed it to `digital.cabinet-office`, which seems most appropriate. I don't think we intentionally send emails to these addresses.